packages/plugin-ext/src/main/browser/main-context.ts

```diff
@@ -34,6 +37,12 @@ export function setUpPluginApi(rpc: RPCProtocol, container: interfaces.Container
     // @ts-ignore
     const windowStateMain = new WindowStateMain(rpc);
 
+    const workspaceService: WorkspaceService = container.get(WorkspaceService);
+
+    // tslint:disable-next-line:no-unused-variable
+    // @ts-ignore
+    const workspaceMain = new WorkspaceMain(rpc, workspaceService);
```

Comment:
do we need to declare the variable ? if not used

Reply:
Removed. Thanks.